test/remote.jl

```diff
@@ -0,0 +1,25 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
Not me to self: next, factor out the wstring change.

On Friday, June 10, 2016, Tony Kelman <notifications@github.com> wrote:

> In test/remote.jl
> <https://github.com/JuliaLang/julia/pull/16590#discussion_r66655948>:
>
> > @@ -0,0 +1,25 @@
> > +# This file is a part of Julia. License is MIT: http://julialang.org/license
>
> I think you got this rebase the wrong way around
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaLang/julia/pull/16590/files/cf5d717e27e183d9ffb040a2b2113466ec472541..50c6cb95371a34f7c2084129517454c9adc709e0#r66655948>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJX_L5lV1vMwLJnqpt-Olkk35O3_Zdzks5qKaa3gaJpZM4InB5w>
> .
>